xbmc/cores/AudioEngine/Utils/AERingBuffer.h

```diff
@@ -285,11 +273,11 @@ class AERingBuffer {
     m_iRead+=size;
   }
 
-  unsigned int m_iReadPos;
-  unsigned int m_iWritePos;
-  unsigned int m_iRead;
-  unsigned int m_iWritten;
-  unsigned int m_iSize;
-  unsigned int m_planes;
+  unsigned int m_iReadPos = 0;
+  unsigned int m_iWritePos = 0;
+  unsigned int m_iRead = 0;
+  unsigned int m_iWritten = 0;
+  unsigned int m_iSize = 0;
+  unsigned int m_planes = 0;
```

Comment:
Shouldn't we also default init `m_Buffer` with `nullptr` here? (line 282)

Reply:
fixed manually
`modernize-use-default-member-init` check didn't convert `NULL` and either I didn't run clang-tidy-diff with `modernize-use-nullptr` check on darwin or it didn't work.